fcrepo-clients/src/main/java/edu/unc/lib/dl/fedora/ClientUtils.java

```diff
@@ -44,6 +44,7 @@
 
 import edu.unc.lib.dl.xml.StandaloneDatastreamOutputFilter;
 
+@SuppressWarnings("deprecation")
```

Comment:
Fixed, but I did notice these deprecated dependencies are also present: 
org.apache.xml.serialize.OutputFormat
org.apache.xml.serialize.XMLSerializer

Maybe we should think about removing them at some point?